autoload/neomake.vim

```diff
@@ -578,3 +594,28 @@ function! neomake#CompleteMakers(ArgLead, CmdLine, CursorPos)
                     \ "v:val =~? '^".a:ArgLead."'")
     endif
 endfunction
+
+function! neomake#WithCallback(file_mode, enabled_makers, ...)
+    let options = a:0 ? { 'exit_callback': a:1 } : {}
```

Comment:
Won't this apply `exit_callback` to every maker in `enabled_makers`. So if you do: `call neomake#WithCallback(0, ['a', 'b', 'c'])` then `exit_callback` will be called three times, right?